src/com/lftechnology/java_training/realworldproblem/VariousCarTester.java

```diff
@@ -0,0 +1,134 @@
+package com.lftechnology.java_training.realworldproblem;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class performs various feature and functionalities of {@link ElectricCar} and {@link PetrolCar}
+ * 
+ * @author Utsab Neupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class VariousCarTester {
+	private static Logger LOGGER = Logger.getLogger(ElectricCar.class.getName());
+	private static Scanner input = new Scanner(System.in);
+	private static int choiceInput = 0;
+
+	public static void main(String[] args) {
+		String inputChoice = "";
+		do {
+			LOGGER.log(Level.INFO,
+					"\n****Simulation of Cars: ****** \n1.Simulation of Electric Car \n2. Simulation of Petrol Car \n3.to exit");
+			do {
+				inputChoice = input.next();
+
+			} while (!isInt(inputChoice));
+			choiceInput = Integer.parseInt(inputChoice);
+			switch (choiceInput) {
+			case 1:
+				ElectricCar teslaCar = new ElectricCar();
```

Comment:
I think its good to assign the object of the ElctricCar to its super Class Car. that way you could make use of polymorphism.